base/range.jl

```diff
@@ -47,23 +47,43 @@ function _colon(start::T, step, stop::T) where T
 end
 
 """
-    range(start[, stop]; length, stop, step=1)
-
-Given a starting value, construct a range either by length or from `start` to `stop`,
-optionally with a given step (defaults to 1, a [`UnitRange`](@ref)).
-One of `length` or `stop` is required.  If `length`, `stop`, and `step` are all specified, they must agree.
```

Comment:
> If `length`, `stop`, and `step` are all specified, they must agree.

`length`, `stop`, and `step` cannot be all specified:

https://github.com/JuliaLang/julia/blob/01d89c66d17c9e6256609f7904979442ed96da7b/base/range.jl#L166-L167
```julia
_range(start::Real, step::Real, stop::Real, length::Integer) = # range(a, step=s, stop=b, length=l)
    throw(ArgumentError("Too many arguments specified; try passing only one of `stop` or `length`"))
```